cpp/modmesh/toggle/profile.cpp

```diff
@@ -36,31 +36,46 @@ std::string TimeRegistry::detailed_report() const
     std::ostringstream ostm;
     /// Header
     ostm
-        << std::setw(40) << total_call_count()
+        << std::setw(50) << total_call_count()
         << " function calls in " << total_time()
         << " seconds" << std::endl;
     ostm
         << std::endl
-        << std::setw(40) << "Function Name"
-        << std::setw(25) << "Call Count"
-        << std::setw(25) << "Total Time (s)"
-        << std::setw(25) << "Per Call (s)"
-        << std::setw(25) << "Cumulative Time (s)"
-        << std::setw(25) << "Per Call (s)"
+        << std::setfill('-') << std::setw(135) << "";
+    ostm
+        << std::endl
+        << std::setw(4)
+        << "|    Function Name    "
+        << "|    Call Count    "
+        << "|    Total Time (s)    "
+        << "|    Per Call (s)    "
+        << "|    Cumulative Time (s)    "
+        << "|    Per Call (s)    |"
+        << std::endl;
+    ostm
+        << std::setfill('-') << std::setw(135) << ""
         << std::endl;
 
     /// Body
     for (auto it = m_entry.begin(); it != m_entry.end(); ++it)
     {
         ostm
-            << std::setw(40) << it->first
-            << std::setw(25) << it->second.count()
-            << std::setw(25) << it->second.time()
-            << std::setw(25) << it->second.time() / it->second.count()
-            << std::setw(25) << it->second.ctime()
-            << std::setw(25) << it->second.ctime() / it->second.count()
+            << std::fixed << std::setprecision(6)
+            << std::setw(4)                            << "|        " 
+            << it->first                               << "          |        "
+            << it->second.count()                      << "         |       "
+            << it->second.time()                       << "       |      "
+            << it->second.time() / it->second.count()  << "      |          "
+            << it->second.ctime()                      << "         |      "
+            << it->second.ctime() / it->second.count() << "      |"
```

Comment:
Add dividers between numbers of each columns

Reply:
Maybe we can have a better way to deal with `"          |        "`. This looks hard to maintain.

Reply:
I don't like the table frames.  It makes parsing impossible.  Please avoid table frames at all.

Reply:
@tigercosmos  Got it. If there are similar tasks in the future, I'll use another way to represent it. But as @yungyuc  said, the frame makes it hard to parse, so I'll remove the frame this time.